<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>131</width>
     <height>158</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="groupBox_rad1">
      <property name="text">
       <string>groupBox1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="groupBox_rad2">
      <property name="text">
       <string>groupBox2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="groupBox_rad3">
      <property name="text">
       <string>groupBox3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="groupBox_rad4">
      <property name="text">
       <string>groupBox4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
